File: sample-video-thumbnail/src/main/java/me/xiaopan/sketch_video_thumbnail_sample/SketchInitializer.java
package me.xiaopan.sketch_video_thumbnail_sample;

import android.content.Context;
import android.support.annotation.NonNull;

import me.xiaopan.sketch.Configuration;
import me.xiaopan.sketch.Initializer;

public class SketchInitializer implements Initializer {

    @Override
    public void onInitialize(@NonNull Context context, @NonNull Configuration configuration) {
        configuration.getUriModelRegistry().add(new VideoThumbnailUriModel());
    }
}

File: sample-video-thumbnail/src/main/java/me/xiaopan/ssvt/SketchInitializer.kt
package me.xiaopan.ssvt

import android.content.Context

import me.xiaopan.sketch.Configuration
import me.xiaopan.sketch.Initializer

class SketchInitializer : Initializer {

    override fun onInitialize(context: Context, configuration: Configuration) {
        configuration.uriModelManager.add(VideoThumbnailUriModel())
    }
}
